src/plugin.js

```diff
@@ -16,21 +16,61 @@ require(['gitbook'], function (gitbook) {
         }) {
         var extraParam = type === "watch" ? "&v=2" : "";
         return `<a class="btn pull-right hidden-mobile" aria-label="github">
-        <iframe style="display:inline-block;vertical-align:middle;" src="https://ghbtns.com/github-btn.html?user=${user}&repo=${repo}&type=${type}&count=${count}&size=${size}${extraParam}" frameborder="0" scrolling="0" width="${width}px" height="${height}px"></iframe>
+            <iframe
+                style="display:inline-block;vertical-align:middle;"
+                src="https://ghbtns.com/github-btn.html?user=${user}&repo=${repo}&type=${type}&count=${count}&size=${size}${extraParam}"
+                frameborder="0"
+                scrolling="0"
+                width="${width}px"
+                height="${height}px"
+            ></iframe>
         </a>`;
     }
 
-
-    function insertGitHubLink({
+    function createUserButton({
         user,
-        repo,
-        types,
         size,
         width,
         height,
         count
         }) {
-        types.reverse().forEach(type => {
+        return `<a class="btn pull-right hidden-mobile" aria-label="github">
+            <iframe
+                style="display:inline-block;vertical-align:middle;"
+                src="https://ghbtns.com/github-btn.html?user=${user}&type=follow&count=${count}&size=${size}"
+                frameborder="0"
+                scrolling="0"
+                width="${width}px"
+                height="${height}px"
+            ></iframe>
+        </a>`;
+    }
+
+    function insertGitHubLink(button) {
+        var {
+            user,
+            repo,
+            type,
+            size,
+            width,
+            height,
+            count
+        } = button;
+
+        var size = size || "large";
+        var width = width || (size === "large" ? "150" : "100");
+        var height = height || (size === "large" ? "30" : "20");
+        var count = typeof count === "undefined" ? "true" : "false";
```

Comment:
We should treat `count: true` setting as `var count = true`.

Can you change this to following?

```js
var count = typeof count === "boolean" ? count : "false";
```

Reply:
This is moved from your code: https://github.com/azu/gitbook-plugin-github-buttons/blob/master/src/plugin.js#L58 
and would be applied to both ``createButton`` and ``createUserButton``. Should I?

Reply:
> This is moved from your code:

Oh, sorry.

> would be applied to both createButton and createUserButton. Should I?

Yes, Could you please do it?

Reply:
BTW, I think ``var count = typeof count === "boolean" ? count : false;`` would be more directly.

One more thing, should we regard non boolean values as ``false`` or ``true``?

Reply:
> One more thing, should we regard non boolean (including undefined) values as false or true?

https://ghbtns.com/github-btn.html?user=twbs&repo=bootstrap&type=watch
Default value seem to be `false`.

Reply:
> var count = typeof count === "boolean" ? count : false;

I agree. (also need to update README.md)